<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>seyahatbilgisiverigirisformu</class>
 <widget class="QWidget" name="seyahatbilgisiverigirisformu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seyahat Bilgisi</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Kaynaklar.qrc">
    <normaloff>:/Resimler/Resimler/seyahat.png</normaloff>:/Resimler/Resimler/seyahat.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="lbIcon">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../Kaynaklar.qrc">:/Resimler/Resimler/seyahat.png</pixmap>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lbSeyahatBilgisi">
         <property name="text">
          <string>Seyahat Bilgisi</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="1">
          <widget class="QDateTimeEdit" name="dteTarihSaat"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="leIkram"/>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spnPeronNo"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbTarih">
           <property name="text">
            <string>Tarih/Saat</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lbTesis">
           <property name="text">
            <string>Tesis</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lbPeronNo">
           <property name="text">
            <string>Peron No</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPlainTextEdit" name="pteTesis"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbIkram">
           <property name="text">
            <string>İkram</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="1">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QPushButton" name="pbIptal">
           <property name="text">
            <string>İptal</string>
           </property>
           <property name="icon">
            <iconset resource="../../Kaynaklar.qrc">
             <normaloff>:/Resimler/Resimler/iptal.png</normaloff>:/Resimler/Resimler/iptal.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pbKaydet">
           <property name="text">
            <string>Kaydet</string>
           </property>
           <property name="icon">
            <iconset resource="../../Kaynaklar.qrc">
             <normaloff>:/Resimler/Resimler/kaydet.png</normaloff>:/Resimler/Resimler/kaydet.png</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spnPeronNo</tabstop>
  <tabstop>dteTarihSaat</tabstop>
  <tabstop>leIkram</tabstop>
  <tabstop>pteTesis</tabstop>
  <tabstop>pbKaydet</tabstop>
  <tabstop>pbIptal</tabstop>
 </tabstops>
 <resources>
  <include location="../../Kaynaklar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbIptal</sender>
   <signal>clicked()</signal>
   <receiver>seyahatbilgisiverigirisformu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
